docs/api/datasets.rst

```diff
@@ -195,8 +195,8 @@ Non-geospatial Datasets
 
 :class:`NonGeoDataset` is designed for datasets that lack geospatial information. These datasets can still be combined using :class:`ConcatDataset <torch.utils.data.ConcatDataset>`.
 
-.. csv-table:: C = classification,  R = regression, S = semantic segmentation, I = instance segmentation, T = time series, CD = change detection, OD = object detection, IC = image captioning
-   :widths: 15 7 15 20 12 11 12 15 13
+.. csv-table:: C = classification,  R = regression, S = semantic segmentation, I = instance segmentation, T = time series, CD = change detection, OD = object detection, IC = image captioning, SR = super resolution
+   :widths: 14 6 14 18 11 10 11 14 12 12
```

Comment:
These widths are for the table, not for the caption. Undo the changes to the widths and you should be fine.